ci/update-deps.js

```diff
@@ -4,9 +4,12 @@ const { default: axios } = require('axios');
 const fs = require('fs');
 const { execSync } = require('child_process');
 const { compareVersions } = require( './utils' );
+const marked = require('marked');
 
 const CONFIG_FILE = './config.json';
 
+const LOBBY_VIP_TOKEN = process.env.LOBBY_VIP_TOKEN;
```

Comment:
Just in case: there is a shorter way, destructured assignment:

```js
const { LOBBY_VIP_TOKEN } = process.env;
```

It also works for arrays.